resources/js/app.tsx

```diff
@@ -74,7 +75,9 @@ void createInertiaApp({
                             <MetaMaskContextProvider>
                                 <TransactionSliderProvider>
                                     <DarkModeContextProvider>
-                                        <App {...props} />
+                                        <Erc1155ContextProvider>
+                                            <App {...props} />
+                                        </Erc1155ContextProvider>
```

Comment:
just something to think about but wondering if we can get rid of this provider and move the functionality to the `AuthContextProvider` since seems to me something related to the session wallet

WDYT?